src/components/directory-content.jsx

```diff
@@ -875,87 +505,42 @@ const DirectoryContent = () => {
         );
     };
 
+    const onGridReady = useCallback(({ api }) => {
+        api?.sizeColumnsToFit();
+    }, []);
+
+    const getRowId = useCallback((params) => params.data.elementUuid, []);
+    const getRowStyle = useCallback((cellData) => {
+        const style = { fontSize: '1rem' };
+        if (
+            ![ElementType.CASE, ElementType.PARAMETERS].includes(
+                cellData.data?.type
+            )
+        ) {
+            style.cursor = 'pointer';
+        }
+        return style;
+    }, []);
+
```

Comment:
can all those methods be created out of the component instead ? It would prevent the usage of useCallback

Reply:
I externalized those in a CustomAGGgrid wrapper dedicated for the directory content, it's called DirectoryContentTable.
It reduces boilerplate code inside directory content imo, wdyt ?